.changeset/breezy-coats-count.md

```diff
@@ -0,0 +1,5 @@
+---
+"octopus-deploy": major
```

Comment:
I'm ok with this. @liam-mackie ?

Reply:
This should be a minor bump IMO, as there is nothing incompatible with the current version.